### DISCUSSION TITLE
[V5] Error with Radio button on iOS device

### DISCUSSION THREAD

**Discussion by foxbunlong** [ant-design/ant-design#42195] [Q&A]
![z4322694073842_2bc6170d9d6b7e7c9089276d76ac7be4](https://user-images.githubusercontent.com/3215633/236621501-3d74cf82-b334-4f07-a620-e284b0b8fc77.jpg)
Please help to fix this on iOS devices. I found it appeared wrongly on iOS device. Thanks

**Comment by yoyo837** [comment]
Duplicate of #42154?

**Reply by foxbunlong** [reply]
No, it happens on Antd homepage website. My code has the same problem and I've already added
`import { StyleProvider } from '@ant-design/cssinjs';`
to wrap my App component

**Reply by yoyo837** [reply]
I think It is duplicate of #42154, See https://ant.design/docs/react/compatible-style#compatible-adjustment

**Reply by yoyo837** [reply]
What's your iOS version?

**Reply by foxbunlong** [reply]
ios 14.4.1
I think it's quite old

**Reply by yoyo837** [reply]
<img width="1216" alt="image" src="https://user-images.githubusercontent.com/6134547/236626275-c10043d3-4a10-4c0e-b1ba-91473a6186d8.png">

**Reply by foxbunlong** [reply]
![image](https://user-images.githubusercontent.com/3215633/236627215-03b2a877-69d3-4f70-9b9f-0fb385bf9710.png)
![z4322976095913_e8d0eedc884d4cfc1c79f249a7edde34](https://user-images.githubusercontent.com/3215633/236627267-7b8b8e30-51bd-4edd-9ba1-bfdc1d893635.jpg)
yes I did add the code as screenshot but my page is wrong. Please help on this

**Reply by yoyo837** [reply]
Can you inspect it with web tools if the css property incompatibility is causing the style issue?
